blueprints/smart/files/__root__/__routepath__/__route__/__dumb__/__name__/__name__.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+type Props = {
```

Comment:
It is flow and I did recently remove it just to slim things down. We'll have to adjust accordingly.